docs/configuration/miscellaneous.md

```diff
@@ -80,6 +80,17 @@ changes in the database indefinitely.
 
 ---
 
+## CHANGELOG_SKIP_EMPTY_CHANGES
+
+Default: False
```

Comment:
How do we feel about having this default to True? IMO it makes more sense to enable by default, as most people won't have any reason to enable it.